src/zimscraperlib/zim.py

```diff
@@ -100,11 +100,26 @@ def to_zimwriterfs_args(
         return arg_list
 
 
-def make_zim_file(build_dir, output_dir, zim_fname, zim_info):
-    """ runs zimwriterfs """
+def make_zim_file(build_dir, output_dir, zim_fname, zim_info, **kwargs):
+    """ runs zimwriterfs 
+    
+    The function takes in the following arguments
+        build_dir: The directory to convert into a ZIM file
+        output_dir: The directory where the ZIM file would be saved
+        zim_fname: The name of the ZIM file
+        zim_info: An object of class ZimInfo
+    
+    Also supports the following keyword arguments that are passed to to_zimwriterfs_args()
+        verbose: Use verbose mode while creating ZIM
+        inflateHtml: Whether try to inflate HTML files
+        uniqueNamespace: Put everything in namespace 'A'
+        withoutFTIndex: No fulltext index
+        minChunkSize: Number of bytes per ZIM cluster
+        redirects: Path to file containing redirects
+    """
     args = (
-        [ZimInfo.zimwriterfs_path]
-        + zim_info.to_zimwriterfs_args()
+        [zim_info.zimwriterfs_path]
```

Comment:
We could use this PR to remove this `zimwriterfs_path` non-sense and just replace it with `["/usr/bin/env", "zimwriterfs"]`